libpromises/files_names.c

```diff
@@ -589,6 +623,35 @@ bool CompressPath(char *dest, size_t dest_size, const char *src)
 
 /*********************************************************************/
 
+char *GetAbsolutePath(const char *path)
+/**
+ * WARNING: Remember to free return value.
+ **/
```

Comment:
Function comments go in doxygen-like style before, not after the function title. You may also want to be a bit more verbose about what this function does. 

Also applies to the other function.

Reply:
Done.